addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -581,20 +599,13 @@ var ListController = BasicController.extend({
      * @returns {Promise}
      */
     _toggleArchiveState: async function (archive) {
-        let resIds;
-        let displayNotif = false;
-        const state = this.model.get(this.handle, {raw: true});
-        if (this.isDomainSelected) {
-            resIds = await this._domainToResIds(state.getDomain(), session.active_ids_limit);
-            displayNotif = (resIds.length === session.active_ids_limit);
-        } else {
-            resIds = this.model.localIdsToResIds(this.selectedRecords);
-        }
+        const resIds = await this.getSelectedIdsWithDomain()
         await this._archive(resIds, archive);
-        if (displayNotif) {
+        const total = this.model.get(this.handle, {raw: true}).count;
+        if (resIds.length === session.active_ids_limit && resIds.length < total) {
```

Comment:
There is now an additional check, in case the total number of records is equal to the session active_ids_limit

Reply:
yep, but this is only necessary when `this.isDomainSelected` is true